src/main/java/com/minecolonies/coremod/items/ItemSupplyCampDeployer.java

```diff
@@ -155,13 +156,20 @@ public static boolean canCampBePlaced(
         final int sizeX = Settings.instance.getActiveStructure().getSizeX();
         final int sizeZ = Settings.instance.getActiveStructure().getSizeZ();
 
+        int groundLevel = zeroPos.getY();
+        final BlockPos groundLevelPos = BlueprintTagUtils.getFirstPosForTag(Settings.instance.getActiveStructure(), "groundlevel");
+        if (groundLevelPos != null)
+        {
+            groundLevel = groundLevelPos.getY();
+        }
+
```

Comment:
As previously noted, you could just call `Settings.instance.getGroundOffset()` instead of repeating this search.